File: ismael032/Questao08.java
import java.util.Scanner;

public class Questao08 {
    public static void main (String [] args) {
	Scanner s;
	double a;
	double b;
	double c;
	double d;
	double e;
	double f;
	double g;
	double h;


	s = new Scanner (Scanner.in);
	System.out.println("** volume do cubo**");
	System.out.println(" digite o lado do cubo ");
	a = s.nextDouble ();
	b = a * a * a;
	System.out.println(" o volume do cubo sera " + b);

	System.out.println("** volume do paralelepipado**");
	System.out.println(" digite a altura do paralelepipado ");
	c = s.nextDouble ();
	System.out.println(" digite o comprimento do paralelepipado");
	d = s.nextDouble ();
	System.out.println(" digite a base do paralelepipado");
	e = s.nextDouble ();
	f = c * d * e;
	System.out.println(" o volume do paralelepipado " + f );

	System.out.println("** volume das esfera **");
	System.out.println(" informe o rai da esfera : ");
	g = s.nextDouble ();
	h = 4 * 3.14 * g * g * g /3;
    }
}

File: pedro019/teste_fluxogramal/programação_basica/questao08.py
print('** volume do cubo **')
lado = float(input('digite o lado do cubo: '))
volume_cubo = lado * lado * lado
print('o volume do cubo será de ' + str(volume_cubo))

print('** volume do paralelepipedo **')
altura = float(input('digite a altura do paralelepipedo: '))
comprimento = float(input('digite o comprimento do paralelepipedo: '))
base = float(input('digite a base do paralelepipedo: '))
volume_paralelepipedo = altura * comprimento * base
print('o volume do paralelepipedo sera de ' + str(volume_paralelepipedo))

print('** volume de uma esfera **')
raio = float(input('digite o raio da esfera: '))
volume_esfera = 4 * 3.14 * raio * raio *raio/3
print('o volume da esfera sera de ' + str(volume_esfera))

